lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -61,18 +61,33 @@ class ReportRenderer {
    */
   _renderReportHeader(report) {
     const header = this._dom.cloneTemplate('#tmpl-lh-heading', this._templateContext);
+    const scoreContainer = this._dom.cloneTemplate('#tmpl-lh-score-container', this._templateContext);
+    const toolbarEl = this._dom.find('.lh-toolbar', header);
+    /** @type {HTMLDivElement} */ (toolbarEl.parentNode).insertBefore(scoreContainer, toolbarEl);
```

Comment:
can the template just have a slot for the score container?

Reply:
sg. replaceChild ftw